auth-api/migrations/versions/b22ee0cd18ad_productsubscriptionsstatus_added.py

```diff
@@ -0,0 +1,88 @@
+"""ProductSubscriptionsStatus added
+
+Revision ID: b22ee0cd18ad
+Revises: 379f1d6b24d4
+Create Date: 2021-04-01 00:58:21.842283
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.sql import column, table
+from sqlalchemy import Boolean, String
+
+
+# revision identifiers, used by Alembic.
+revision = 'b22ee0cd18ad'
+down_revision = '379f1d6b24d4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('product_codes', sa.Column('default_subscription_status', sa.String(length=30), nullable=False,
+                                             server_default='ACTIVE'))
+
+    op.create_table('product_subscriptions_statuses',
+                    sa.Column('created', sa.DateTime(), nullable=True),
+                    sa.Column('modified', sa.DateTime(), nullable=True),
+                    sa.Column('code', sa.String(length=30), nullable=False),
+                    sa.Column('description', sa.String(length=100), nullable=True),
+                    sa.Column('default', sa.Boolean(), nullable=False),
+                    sa.Column('created_by_id', sa.Integer(), nullable=True),
+                    sa.Column('modified_by_id', sa.Integer(), nullable=True),
+                    sa.ForeignKeyConstraint(['created_by_id'], ['users.id'], ),
+                    sa.ForeignKeyConstraint(['modified_by_id'], ['users.id'], ),
+                    sa.PrimaryKeyConstraint('code')
+                    )
+
+    # add master data
+
+    product_subscriptions_statuses_table = table('product_subscriptions_statuses',
+                                                 column('code', String),
+                                                 column('description', String),
+                                                 column('default', Boolean)
+                                                 )
+    op.bulk_insert(
+        product_subscriptions_statuses_table,
+        [
+            {'code': 'REJECTED', 'description': 'Status for a rejected account',
```

Comment:
instead of account, can you add the description as product.

Reply:
fixed